.golangci.yml

```diff
@@ -79,6 +79,7 @@ linters-settings:
       - '^github\.com/neondatabase/autoscaling/neonvm/apis/neonvm/v1\.VirtualMachine(Migration)?(Spec)?$'
       - '^github\.com/neondatabase/autoscaling/neonvm/apis/neonvm/v1\.IPPool$'
       - '^github\.com/neondatabase/autoscaling/pkg/agent/core\.ActionSet$'
+      - '^github\.com/neondatabase/autoscaling/pkg/agent/core\.Action.*$'
```

Comment:
Why?

(if it's to reduce the diff on the tests, I would rather be explicit there)